bin/run.sh

```diff
@@ -13,8 +13,8 @@
 #LOG_LEVEL=10 \
 #PORT=7891 \
 
-# nodemon -x --harmony iojs app.js
+# nodemon -x node app.js
 
-    # nodemon --harmony app.js
+    # nodemon app.js
```

Comment:
These are commented out. I'm leaving this scripts as they were apart from changing to reflect that we are using node 4.2.X now.